snap/snapcraft.yaml

```diff
@@ -69,6 +71,7 @@ parts:
 
   device-modbus:
     source: .
+    source-tag: v1.1.1
```

Comment:
hmm I don't think this is right. We don't want to set what version of the local repo this is being built from in the git tree, we instead to do a v1.1.1 release of this repo, we would instead run the snap build jobs against the version of the snapcraft.yaml that was in v1.1.1.

Anyways, this probably won't do the expected thing since we have `source: .` which is not a git URL (and we also don't specify `source-type: git`). I'm surprised that snapcraft didn't complain about this, since we don't have a git source, we are just using a local directory source and there's no way for snapcraft to know how to checkout the v1.1.1 tag if it's just a directory (i.e. it could be a bzr tree and not git). Maybe snapcraft is smart enough to introspect this, but this is confusing as is.